File: LongestConsecutiveSequence.java
class Solution {
    public int longestConsecutive(int[] nums) {
        if(nums.length == 0){
            return 0;
        }
        Arrays.sort(nums);
        int max = 0;
        int count = 1;
        for(int i = 1; i < nums.length; i++){
            if(nums[i-1] == (nums[i]-1) || nums[i-1] == (nums[i])){
                if(nums[i-1] != (nums[i])){
                    count++;
                }
            }
            else{
                if(count  > max){
                    max = count;
                }
                count = 1;
            }
        }
        if(count > max){
            max = count;
        }
        return max;
    }
}

File: LongestConsecutiveSequence.py
class Solution(object):
    def longestConsecutive(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        if len(nums) == 0:
            return 0
        nums.sort()
        count = 1
        maxcount = 0
        for i in range(1, len(nums)):
            if nums[i] != nums[i-1]:
                if nums[i] != nums[i-1]+1:
                    maxcount = max(maxcount, count)
                    count = 1 
                else:
                    count += 1
        return max(maxcount, count)
        